tests/phpunit/tests/media.php

```diff
@@ -3278,6 +3278,78 @@ public function test_wp_get_attachment_image_loading_opt_out_individual() {
 		$this->assertStringNotContainsString( ' loading=', $img );
 	}
 
+	/**
+	 * @ticket 57086
+	 *
+	 * @dataProvider data_wp_get_attachment_image_decoding_attr
+	 *
+	 * @covers ::wp_get_attachment_image
+	 */
+	public function test_wp_get_attachment_image_decoding_attr( $decoding, $expected ) {
+		if ( 'no value' === $decoding ) {
+			$image = wp_get_attachment_image( self::$large_id, 'thumbnail', false, array() );
+		} else {
+			$image = wp_get_attachment_image( self::$large_id, 'thumbnail', false, array( 'decoding' => $decoding ) );
+		}
+
+		$this->assertSame( $expected, $image );
```

Comment:
We shouldn't use a concrete `$expected` value, as this creates problems in the unit tests (see failures) and is generally unnecessary since all we want to check is presence of the `decoding` attribute.

I think we should change this `assertSame` to rely on `assertStringContainsString` / `assertStringNotContainsString`.

Reply:
Updated in https://github.com/WordPress/wordpress-develop/pull/3633/commits/10fcec3d6860702e1e5e77cf9b10f3cf431ab0b5.